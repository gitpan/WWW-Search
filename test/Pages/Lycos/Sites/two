http://lycospro.lycos.com/cgi-bin/pursuit?adv=1&cat=dirw&matchmode=or&maxhits=100&mtemp=sites&query=Fett
http://lycospro.lycos.com/cgi-bin/pursuit?adv=1&cat=dirw&matchmode=or&maxhits=10&mtemp=sites&query=Fett&first=11&u2=N&u1=N&u3=Y&u0=N
