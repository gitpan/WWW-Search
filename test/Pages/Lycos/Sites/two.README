This set of test-result pages was created on Tue Oct 12 09:41:31 1999
