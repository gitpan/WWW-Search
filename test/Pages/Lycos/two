http://lycospro.lycos.com/cgi-bin/pursuit?adv=1&matchmode=or&maxhits=100&mtemp=nojava&query=Vader+and+choke
http://lycospro.lycos.com/cgi-bin/pursuit?cat=lycos&first=101&maxhits=100&query=Vader+and+choke&matchmode=or&adv=1&mtemp=nojava
