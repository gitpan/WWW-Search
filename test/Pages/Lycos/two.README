This set of test-result pages was created on Mon Aug 16 16:29:44 1999
