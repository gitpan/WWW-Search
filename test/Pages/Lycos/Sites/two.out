http://www.arteartarte.com/english/dora.htm
http://194.7.253.55/dsifgkomr.html
http://www.sedin.org/propeng/bib_dic.htm
http://194.7.253.55/dsifteresa.html
http://www.eskimo.com/~recall/bleed/0416.htm
http://tigerden.com/%7Eberios/lba.html
http://www.andrew.cmu.edu/~ctb/anarchy/docs/Sabatini.lib.txt
http://www.spunk.org/library/otherpol/critique/sp000713.txt
http://members.aol.com/gsvloc/ida_glo2.htm
http://anxiety-closet.mit.edu:8001/activities/libertarians/libertarians/Essays/Libertarianism-Bogus-Anarchy
http://www.al-bushra.org/arbhrtg/arbxtn01.htm
http://194.7.253.55/dsiftlano.html
http://www.geology.wisc.edu/~delitsin/chapek
http://abbey.apana.org.au/History/Hughes/par1943.htm
http://abbey.apana.org.au/HISTORY/HUGHES/par206.htm
http://www.eco.utexas.edu/faculty/Cleaver/sect5.html
http://home.talkcity.com/PicassoPl/wsgarbi/index.html
http://tekka.wwa.com/~mjm/almanac2/december/1226.html
http://www.abbc.com/aaargh/engl/BLantisem7.html
http://h-net2.msu.edu/~habsweb/newmembers.html
http://tekka.wwa.com/~mjm/almanac2/september/0904.html
http://mercury.panola.cc.tx.us/library/ENG/index/t000026.htm
http://www.eco.utexas.edu/faculty/Cleaver/sect2.html
http://www.csn.net/advent/cathen/03113a.htm
http://www.cios.org/getfile\MASSCOMM_NB126
http://www.cios.org/getfile\MASSCOMM_NB131
http://www.asiatica.org/publications/jsawsbookabs.asp
http://math.idbsu.edu/gas/other_gilbert/princess.txt
